reference/7/Microsoft.PowerShell.Utility/Test-Json.md

```diff
@@ -101,7 +120,9 @@ Accept wildcard characters: False
 
 ### -Schema
 
-Specifies a Schema to validate the JSON input against.  If passed `Test-Json` will validate that the Json input conforms to the spec specified by the `-Schema` parameter and return `$True` only if the input conforms to the provided Schema.
+Specifies a Schema to validate the JSON input against. If passed `Test-Json` will validate that the
+Json input conforms to the spec specified by the **Schema** parameter and return `$True` only if the
+input conforms to the provided Schema.
```

Comment:
Should we expand to say that `$False` details the failure (such as `IntegerExpected: #/age`)?